src/gui/sdlmain.cpp

```diff
@@ -50,7 +50,7 @@
 #include "cross.h"
 #include "control.h"
 
-#define MAPPERFILE "mapper-" VERSION ".map"
+#define MAPPERFILE "mapper-sdl2-" VERSION ".map"
```

Comment:
The format seems the same, but the key numbering changed completely. Perhaps it would be possible to re-create mapping from SDL2 to SDL1-compatible... but I don't think the effort is worth it.

Reply:
Sounds good - was just curious (and makes sense!).
I also have never used the mapper for the same reason you've mentioned.